### PR TITLE
De120 de130

### PR DESCRIPTION
**Pull request by jolf**
Fix for the tasks for DE120, DE130, and the task TA1703 for DE333.
I do not know why, but it included Davids previous commits for the JS problem. They should not be part of this review.

### CODE REVIEW

File: config/locales/da.yml [hunk 1]
```diff
@@ -120,6 +123,7 @@ da:
       preservation_state: "Bevaringsstatus"
       preservaton_details: "Bevaringsdetaljer"
       preservation_date: "Bevarings status dato"
+      preservation_initiated_date: "Bevaring startet dato"
```
Comment (jolf) [inline config/locales/da.yml]: Come to think of it, this should actually be: "Seneste bevaring igangsat"

File: app/views/instances/_show_administrative.html.erb [hunk 2]
```diff
@@ -74,15 +74,19 @@
...
-    <strong><%= t('.preservation_date') %></strong>
+    <strong><%= t('instances.show_administrative.preservation_date') %></strong>
   </td>
   <td>
     <span><%= @instance.preservation_modify_date %></span>
   </td>
 </tr>
 <tr>
+  <td><strong><%= t('instances.show_administrative.preservation_initiated_date') %></strong></td>
+  <td><%= @instance.preservation_initiated_date %></td>
```
Comment (jolf) [inline app/views/instances/_show_administrative.html.erb]: Perhaps a default value telling, that it has not been preserved, if this date is nil or empty?

File: app/views/content_files/_file_info.html.erb [hunk 3]
```diff
@@ -22,6 +16,18 @@
...
+  <td><strong><%= t('instances.show_files.bev_warc_id') %></strong></td>
+  <td><%= file.warc_id %></td>
+</tr>
+<tr>
+  <td><strong><%= t('instances.show_files.bev_file_warc_id') %></strong></td>
+  <td><%= file.file_warc_id %></td>
+</tr>
+<tr>
+  <td><strong><%= t('instances.show_files.bev_preservation_initiated_date') %></strong></td>
+  <td><%= file.preservation_initiated_date %></td>
```
Comment (jolf) [inline app/views/content_files/_file_info.html.erb]: Perhaps a default value telling, that it has not been preserved, if this date is nil or empty?

**Comment by jolf** [comment]
It seems that I've made the pull-request into the 'development' branch and not the 'chronos-spring1' branch, which I intended.

**Comment by xenma** [comment]
:+1:

**Pull request closed without merging**